spec/features/user_can_view_past_orders_spec.rb

```diff
@@ -0,0 +1,62 @@
+require 'rails_helper'
+
+RSpec.feature "User can view past orders" do
+  context 'an exisiting user, who has orders' do
+    attr_reader :order1, :order2
+
+    before do
+      user = create(:user)
+      @order1, @order2 = create_list(:order, 2)
+      user.orders << [order1, order2]
+
+      allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(user)
```

Comment:
@Carmer @bethsebian is there anyway to break up this line? Or is that good practice?